pkg/ring/ring.go

```diff
@@ -130,11 +131,16 @@ func New(cfg Config) (*Ring, error) {
 			"Number of ingesters in the ring",
 			[]string{"state"}, nil,
 		),
-		numTokensDesc: prometheus.NewDesc(
-			"cortex_ring_tokens",
+		totalTokensDesc: prometheus.NewDesc(
+			"cortex_ring_tokens_total",
 			"Number of tokens in the ring",
 			nil, nil,
 		),
+		numTokensDesc: prometheus.NewDesc(
+			"cortex_ring_tokens_owned",
+			"The number of tokens in the ring owned by the ingester",
+			[]string{"ingester"}, nil,
+		),
```

Comment:
Can we make these be global and use promauto whilst we're here?

Reply:
Thinking about it, it'll be bad idea to make this global. We need to just expose the active ingesters, not all the ingester throughout the lifetime of the distributor and having it the current way enables us to do it.

Reply:
Fair enough!